src/constants/api/misc.ts

```diff
@@ -0,0 +1,7 @@
+let origin = '/api';
+
+const MiscURI = {
+    sdkVersionURI: `${origin}/misc/sdk/:key`,
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/FeatureProbe/feature-probe-ui/pullRequest?prid=9905084)
```suggestion
  sdkVersionURI: `${origin}/misc/sdk/:key`,
```